modules/admin_manual/pages/release_notes.adoc

```diff
@@ -27,6 +28,71 @@
+
+=== Sharing information in subfolders
+
+ownCloud Server 10.4 puts focus on user awareness for shared areas to prevent accidentally sharing data or changing other users' data as well as to make it easier for users to recognize who has access to shared areas. Practically, users can now better recognize shared resources using a new share overlay indicator on file and folder icons. The indicators are also applied to resources which are not directly shared but are part of a share (when working in a shared folder).
+
+Apart from that, the sharing sidebar panels have been improved to also show users/groups and public links that have access through shares on parent folders. These will be shown as static entries with a "via" indicator that allows users to jump to the parent folder and to change the share properties if desired.
+
+These information are only shown to share owners (users that created shares) as other share recipients are not entitled to get detailed information about who else has access.
+
+=== MariaDB 10.4 and PostgreSQL 10 support
+
+The discontinuation of PHP 7.0 enables support for MariaDB up to version 10.4 and PostgreSQL 10. Server 10.4 is fully tested against these database versions and proven to work stable.
+
+=== Other notable changes
+
+* External storages can now be mounted in read-only mode. https://github.com/owncloud/core/pull/36397[#36397]
+* Filter options (`--enabled` and `--disabled`) have been added to the `occ app:list` command to only show enabled or disabled apps, respectively. https://github.com/owncloud/core/pull/36520[#36520]
+* Support for Oracle DB connection strings has been added to be able to use Oracle-specific configuration settings like fail over. https://github.com/owncloud/core/pull/36489[#36489]
+* Two new config.php options (`blacklisted_files_regex` and `excluded_directories_regex`) have been added to allow to exclude files and folders from ownCloud using regular expressions (e.g., to prevent creating / renaming / scanning certain file types like ".pst"). See config.sample.php for more information. https://github.com/owncloud/core/pull/36360[#36360]
+* Previously, when a settings section did not have any panels to display, an "Error" was shown. This has been changed to be more user friendly. https://github.com/owncloud/core/pull/36776[#36776]
+* The memory consumption of the `occ files:checksums:verify` command has been optimized and the command will now show progress information. https://github.com/owncloud/core/pull/36787[#36787]
+* The memory consumption of the trash bin expiration background job has been optimized. https://github.com/owncloud/core/pull/36565[#36565]
+
+=== Solved known issues
+
+* Folder download via the web interface now works also on MacOS Catalina. https://github.com/owncloud/core/pull/36722[#36722]
+* User creation now allows "+" characters in the user id (e.g., to invite guests with mail addresses containing "+"). https://github.com/owncloud/core/pull/36613[#36613]
```

Comment:
Note: this will require also updating the guests app version, after installing ownCloud 10.4
If you have LDAP, then an update of the `user_ldap` app will also be required.

I don't know if you want to somehow mention that detail in these release notes.